nsxt/resource_nsxt_manager_cluster.go

```diff
@@ -168,7 +168,7 @@ func waitForNodeStatus(d *schema.ResourceData, m interface{}, nodes []NsxCluster
 		log.Printf("[DEBUG]: API probing for NSX is disabled")
 		return nil
 	}
-	connector := getPolicyConnectorForInit(m, false)
+	connector := getStandalonePolicyConnector(m, false)
```

Comment:
I still see the same failure with the PR applied.
I've tried to change this line to 
```
connector := getPolicyConnector(m)
```
Which worked - I think that as with `getStandalonePolicyConnector` we skip license assignment, this is where we want to assign the license and therefore use the regular `getPolicyConnector`

Reply:
This connection just waits for node status, it should not require license, but we don't want it to initialize the version for the main NSX connection and thus skip license assignment for it..